tests/fixtures/test_wrangler_factors_output.json

```diff
@@ -1,51 +1,51 @@
 [
     {
         "imputation_factor_Q601_asphalting_sand": 1,
-        "imputation_factor_Q602_building_soft_sand": 1,
+        "imputation_factor_Q602_building_soft_sand": 1.0,
         "imputation_factor_Q603_concreting_sand": 1,
         "imputation_factor_Q604_bituminous_gravel": 1,
-        "imputation_factor_Q605_concreting_gravel": 1,
+        "imputation_factor_Q605_concreting_gravel": 1.0,
         "imputation_factor_Q606_other_gravel": 1,
-        "imputation_factor_Q607_constructional_fill": 1,
+        "imputation_factor_Q607_constructional_fill": 1.0,
```

Comment:
Is there a reason this became a mix of int and floats now?

Reply:
I've no idea. There was a different set that changed from floats to ints as well but it seems when they are read back in there is no different. I can only assume that because of the indent it does some sort of extra processing that caused it????

Reply:
It should not have done, from what I've seen the options selected are indenting and sort only. Very strange. It's just sample data for testing which passes so I'm going to tick it anyway now.